prov/shm/src/smr_progress.c

```diff
@@ -906,6 +889,8 @@ static void smr_progress_connreq(struct smr_ep *ep, struct smr_cmd *cmd)
 		smr_map_to_region(&smr_prov, ep->region->map, idx);
 		peer_smr = smr_peer_region(ep->region, idx);
 	}
+
+	smr_set_ipc_invalid(ep->region, idx);
```

Comment:
I'd rename this to either `smr_set_ipc_valid` or `smr_set_ipc_support` or something like that. This kind of sounds like we're disabling ipc always